pages/community/precice-workshop-organization.md

```diff
@@ -0,0 +1,116 @@
+- For the talks: good acoustics and visibility, recording infrastructure
+- For the world café: (standing) tables, placed far from each other, isolated from noisy areas of the building
+- For the catering: ideally next to the rooms, isolated, standing
+
+With "recording infrastructure", we mean infrastructure to capture the speaker audio+video, slides, and an additional microphone in separate stream (a meeting camera system is typically not fitting the purpose). We can provide specialized hardware to do the mixing of these channels into what you later see as recordings on YouTube, if this is not already available in the room.
+
+The rooms for the training and the talks are typically different, even if the schedule is typically structured as a single track. The catering and the world café can typically use the same room and tables, but reusing the collaborative working room for the world café is also fine.
+
+### Define financing scheme
+
+Typically, the workshop is funded by participation fees. An important dependency for subsequent steps is which entity can collect and process such participation fees. This might be an association supporting scientific activities, an event management unit of a university, or the organizing institute (e.g., via project infrastructure).
+
+Multiple types of tickets are typically defined:
+
+- early -vs- late registration
+- full price -vs- discount for academia
+- training course included -vs- only main part of the workshop
+
+In addition, free registration is accounted for:
+
+- holders of a support license
+- invited speakers
+- special cases
+
+### Set milestones
+
+Set clear responsible persons for each task, and track progress in a collaborative project board.
+
+Important dates to set:
+
+- "save the date" announcement: one year ahead
+- call for contributions (needs a form): open six months ahead. An early call mainly acts as advertisement.
+- open registration (needs a form): six months ahead. Early registration is important for participants that need a visa application to travel.
+- abstract submission deadline and call acceptance: the earlier the better. Can be after the early-bird registration deadline. The call acceptance can be as short as one week after the deadline.
+- early-bird registration deadline: important for estimating the number of participants, driven by the catering requirements.
+- request catering offers: typically 1-2 months before the workshop.
+- reserve tables for the workshop dinner: typically 1-2 months before the workshop.
+- late registration deadline: typically one week before the workshop, does not affect much.
+
+## Registration infrastructure
+
+Two forms are mainly needed, and it is easier if they are decoupled: an abstract submission form, and a registration form.
+
+The abstract submission form should collect:
+
+- Kind of contribution (talk or poster, if offered)
+- Presenter
+- Affiliation
+- Contact email
+- Title
+- Abstract (plain text, short)
```

Comment:
```suggestion
- Abstract (plain text, max 1500 chars including spaces)
```

Reply:
This sounds too specific and arbitrary to me for a future-proof guide, but I guess we can use it as a guideline of what we mean "short".